src/SparqlLua.php

```diff
@@ -5,10 +5,10 @@
 namespace ProfessionalWiki\SPARQL;
 
 use InvalidArgumentException;
-use MediaWiki\Extension\Scribunto\Engines\LuaCommon\LibraryBase;
 use MediaWiki\MediaWikiServices;
+use Scribunto_LuaLibraryBase;
 
-class SparqlLua extends LibraryBase {
+class SparqlLua extends Scribunto_LuaLibraryBase {
```

Comment:
The LibraryBase class does not exist in MW 1.39. The CS complaint is handled by https://github.com/ProfessionalWiki/SPARQL/pull/14/files#diff-0361f0c81f363476ddc6f44ab36fcbe66ee685d5f4c2a46b054924591544b766.